VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/DeveloperScreen/OverrideApiScreen.tsx

```diff
@@ -0,0 +1,542 @@
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useSelector } from 'react-redux'
+
+import { StackScreenProps } from '@react-navigation/stack'
+
+import { Button } from '@department-of-veterans-affairs/mobile-component-library'
+import _ from 'lodash'
+
+import { Box, FeatureLandingTemplate, SelectorType, TextArea, TextView, VASelector, VATextInput } from 'components'
+import { NAMESPACE } from 'constants/namespaces'
+import { HomeStackParamList } from 'screens/HomeScreen/HomeStackScreens'
+import { RootState } from 'store'
+import { APIError } from 'store/api'
+import { DemoState, updateErrorOverrides } from 'store/slices/demoSlice'
+import { useAppDispatch, useTheme } from 'utils/hooks'
+
+type OverrideAPIScreenProps = StackScreenProps<HomeStackParamList, 'OverrideAPI'>
+
+const APIGroupings = [
```

Comment:
It would be helpful to create a type for this, so that when others developers add endpoints in the future, TypeScript would catch improper formatting

Reply:
I added a type to the APIGroupings to have TypeScript catch formatting